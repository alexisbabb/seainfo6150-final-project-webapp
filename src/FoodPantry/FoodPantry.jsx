import React from 'react'; 
import styles from "./FoodPantry.module.css";

const FoodPantry = (props) => {
    return (
        <div>
            <article className = {styles.pageContent}>
            <header className = {styles.pageTitle} >
            Food for all!
        
            </header>
            <div className = {styles.pageDetails}>
            <img className = {styles.pageImage} src="https://charlottenc.gov/AnimalsCMPD/Donate/PublishingImages/pet%20food.jpg" alt="Food Pantry"/>
                    <ul>
                    <u>Pick-Up:</u> DHAâ€™s Pet Food Pantry was created to help members of our community who are having trouble taking care of their pets. The pantry is operated on the honor system; recipients are not required to show proof of income to determine eligibility. 
                    <br></br>
                    <br></br>
                    <u>Donations:</u> The primary focus of the pantry is collecting food and supplies (e.g. canned and dry food, cat litter) for cats and dogs and distributing them to those in need. We ask that items donated specifically for the pantry be new and unused, and that food containers be unopened. We thank you for your generosity!
                    <br></br>
                    <br></br>
                    <u>Hours:</u> The food pantry is open 9am-5pm on Monday through Thursday or 10am-5pm on Friday through Sunday for both donations and pick-up. 
                    <br></br>
                    </ul>  
            </div>
        </article>

        </div>
    )
}



export default FoodPantry