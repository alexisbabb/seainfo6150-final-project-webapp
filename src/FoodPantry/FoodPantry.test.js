import React from "react";
import FoodPantry from "./FoodPantry"

describe("FoodPantry tests", () => {
    it("renders correctly", () => {
        const src = "https://charlottenc.gov/AnimalsCMPD/Donate/PublishingImages/pet%20food.jpg";
        const { container } = render(<FoodPantry src={src}/>);
        expect(container).toMatchSnapshot();
    });
});