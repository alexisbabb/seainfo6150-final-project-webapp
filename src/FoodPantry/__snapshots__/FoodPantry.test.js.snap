// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodPantry tests renders correctly 1`] = `
<div>
  <div>
    <article
      class="pageContent"
    >
      <header
        class="pageTitle"
      >
        Food for all!
      </header>
      <div
        class="pageDetails"
      >
        <img
          alt="Food Pantry"
          class="pageImage"
          src="https://charlottenc.gov/AnimalsCMPD/Donate/PublishingImages/pet%20food.jpg"
        />
        <ul>
          <u>
            Pick-Up:
          </u>
           DHAâ€™s Pet Food Pantry was created to help members of our community who are having trouble taking care of their pets. The pantry is operated on the honor system; recipients are not required to show proof of income to determine eligibility.
          <br />
          <br />
          <u>
            Donations:
          </u>
           The primary focus of the pantry is collecting food and supplies (e.g. canned and dry food, cat litter) for cats and dogs and distributing them to those in need. We ask that items donated specifically for the pantry be new and unused, and that food containers be unopened. We thank you for your generosity!
          <br />
          <br />
          <u>
            Hours:
          </u>
           The food pantry is open 9am-5pm on Monday through Thursday or 10am-5pm on Friday through Sunday for both donations and pick-up.
          <br />
        </ul>
      </div>
    </article>
  </div>
</div>
`;
